docs/tutorials/tutorial-compaction.md

```diff
@@ -90,7 +90,7 @@ We have included a compaction task spec for this tutorial datasource at `quickst
 
 This will compact all segments for the interval `2015-09-12/2015-09-13` in the `compaction-tutorial` datasource.
 
-The parameters in the `tuningConfig` control how many segments will be present in the compacted set of segments.
+The parameters in the `tuningConfig` gives control of how many maximum rows will be present in each segments, among other tuning configurations.
```

Comment:
```suggestion
The parameters in the `tuningConfig` control the maximum number of rows present in each of the compacted segments and other tuning configurations.
```
@kfaraz , @Vespira what about this clarification?

Reply:
Maybe retain some of the original language too (about the no. of segments) as it connects with the next paragraph.

```suggestion
The parameters in the `tuningConfig` control the maximum number rows present in each compacted segment and thus affect the number of segments created by compaction.
```